dashboard/dashboard/templates/racetrack/administration.html

```diff
@@ -46,10 +46,15 @@ <h5 class="card-title">Infrastructure Targets</h5>
             Available Infrastructure Targets:
             {% if infrastructure_targets|length > 0 %}
                 <ul>
-                {% for infrastructure_target, plugin_manifest in infrastructure_targets.items %}
-                    <li>
-                        <span><b>{{ infrastructure_target }}</b> (by plugin {{ plugin_manifest.name }} {{ plugin_manifest.version }})</span>
-                    </li>
+                {% for infrastructure, instances in infrastructure_instances %}
+                <span>{{ infrastructure }}</span>
+                    <ul>
```

Comment:
I think you missed `<li>` tag as pointed out here https://stackoverflow.com/a/5899394/6772197

> The nested list should be inside a < li > element of the list in which it is nested.